accepted/0000-caasp-integration.md

```diff
@@ -0,0 +1,102 @@
+
+After the user has imported the `kubeconfig`, obtained from CaaS Platform, the process would be:
+
+- When a user registers a new cluster, we introduce a new workflow to bootstrap (using Systems > Bootstrapping) each node of the cluster
+- When every cluster node is registered, a system group with the same name of the cluster is created and all registered systems are moved into this system group
+- Each node presented the Nodes information is a link to the system overview page of that node
+
+## Documentation
+
+In addition to all the above, we are enriching our documentation on the CaaS Platform deployment. In the following sections, we are going to describe what we should introduce in the documentation.
+
+### AutoYaST Deployment documentation
+
+CaaS Platform provides AutoYaST templates for deploying a standard cluster (in the `SUSE-CaaSP-Management` package) comprising of:
+
+- 2 Load Balancers
+- 3 Masters
+- 3 Workers
+
+In the Uyuni/SUSE Manager documentation, we need to provide a step-by-step guide on how to customize the AutoYaST template, import it in Uyuni/SUSE Manager and deploy it to bare-metal machines.
+
+## Next steps
+
+Possible next steps are presented as additional "stretch goals": if time permits, those goals can be achieved in the scope of the initial integration.
+
+### Minion blackout configuration
+
+If the [minion blackout](https://docs.saltstack.com/en/latest/topics/blackout/) is feasible, we can set to blackout every minion registered to Uyuni/SUSE Manager that is part of a Kubernetes or CaaS Platform cluster. Every minion that is in blackout mode will reject all incoming commands.
+
+### Container registry documentation
+
+In the documentation, we can write a guide on how to set up a container registry (using a standard container registry or Portus) and mirror SUSE container registry.
+
+### Management node
+
+As part of the support for the CaaS Platform version 4, SUSE Manager can elect (via the Add-On System Type) a node as "Management Node" for the cluster. This node must run SUSE Linux Enterprise Server and have `skuba` (a tool packaged by CaaS Platform) to deploy and manage the node.
+Alternatively, we can choose to run `skuba` on the SUSE Manager server, we should add it to the packaging pattern of SUSE Manager server. The package is Go-vendored, no other dependency is required.
+
+`skuba` is not packaged for OpenSUSE at the time of writing: Uyuni support will be implemented but enabled later, as soon as the package is released for OpenSUSE.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+If a user does not read the documentation, a Kubernetes cluster can be broken by simply rebooting a node from Uyuni/SUSE Manager.
+
+# Alternatives
+[alternatives]: #alternatives
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+- CaaS Platform: for Container registry do you suggest a standard container registry or Portus?
```

Comment:
Portus _is_ the standard container registry for SLE ;-)

Reply:
Fixed